### PR TITLE
docs: update Kafkacat to Kcat

### PR DESCRIPTION
**Pull request by jbyrne-redhat**
**Notes for reviewers**

This PR includes the following updates:

- Updates the [Kafkacat quick start](https://access.redhat.com/documentation/en-us/red_hat_openshift_streams_for_apache_kafka/1/guide/ee92cfdb-9587-42f8-80d5-54169e0e3c07) to reference Kcat (the new name). This includes all of the commands.
- Updates references to Kafkacat in other MAS guides
- Applies [MAS style updates](https://docs.google.com/document/d/1TERH4XRH6La44hUQH88LUaax-35JH3dLXmceGJoLv7I/edit#) to the quick start

An HTML preview of the updated quick start is available [here](http://file.emea.redhat.com/~jbyrne/JK-9559/README.html).

### CODE REVIEW

**Review by AedinC — APPROVED** [review]
Nice set of updates John. Just had one suggestion.

File: docs/kafka/kcat-kafka/README.adoc [hunk 1]
```diff
@@ -79,134 +79,126 @@ END GENERATED ATTRIBUTES
...
  -X security.protocol=SASL_SSL -X sasl.mechanisms=PLAIN \
  -X sasl.username="$RHOAS_SERVICE_ACCOUNT_CLIENT_ID" \
  -X sasl.password="$RHOAS_SERVICE_ACCOUNT_CLIENT_SECRET" -P
 ----
 
-NOTE: {product-kafka} also supports the SASL/OAUTHBEARER mechanism for authentication, which is the recommended authentication mechanism to use. However, Kafkacat does not yet fully support OAUTHBEARER, so this example uses SASL/PLAIN.
+The example uses the SASL/PLAIN authentication mechanism with the server and credential environment variables that you set previously.
 
-NOTE: A Kafkacat producer might behave differently depending on the operating system. For example, on Mac operating systems, messages are sent by redirecting them to the Kafkacat binary in the format `echo "message" | kafkacat ...`.
+NOTE: The recommended authentication mechanism for {product-kafka} is SASL/OAUTHBEARER, which means that you must also specify an access token. However, Kcat does not yet fully support OAUTHBEARER. Therefore, the preceding example uses SASL/PLAIN authentication.
```
[the remark this answers is not in the document]
Comment (jbyrne-redhat) [reply docs/kafka/kcat-kafka/README.adoc:185]: Good point, @AedinC.

**Comment by jbyrne-redhat** [comment]
Hi, @henryZrncik. This is the PR mentioned in https://issues.redhat.com/browse/MGDSTRM-9869. All other reviews are complete, so it is now ready for final QE verification. Thanks!

**Review by henryZrncik — APPROVED** [review]

**Comment by jbyrne-redhat** [comment]
QE-verified by @henryZrncik. All steps work as documented. Ready to merge and publish.

**Pull request merged**